master/reference/calicoctl/resources/policy.md

```diff
@@ -92,8 +92,10 @@ applied action is deny.
 |-------------|-----------------------------|-------------------|--------|------------|
 | tag (deprecated)      | Match on tag. |  | string | |
 | notTag (deprecated)   | Negative match on tag. |  | string | |
-| net    | Match on CIDR. | Valid IPv4 or IPv6 CIDR  | cidr | |
-| notNet | Negative match on CIDR. | Valid IPv4 or IPv6 CIDR | cidr | |
+| net                   | Deprecated (use "nets" instead): Match on CIDR. | Valid IPv4 or IPv6 CIDR  | cidr | |
+| nets                  | Match any one of the listed CIDRs. | List of valid IPv4 or IPv6 CIDRs  | list of cidr |
+| notNet                | Deprecated (use "notNets" instead): Negative match on CIDR. | Valid IPv4 or IPv6 CIDR | cidr | |
+| nets                  | Negative match on CIDRs. | List of valid IPv4 or IPv6 CIDRs  | list of cidr |
```

Comment:
Agree with 1 & 2 but not sure about 3; the list still needs to contain valid CIDRs.